src/main/java/core/basesyntax/dao/writeReportImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Account;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class writeReportImpl implements writeReport {
+    private Account account = new Account();
```

Comment:
The `Account` object is instantiated within the class. Ensure this is the intended design, as it may be better to pass an `Account` object to the class or method if the account data is meant to be dynamic or provided externally.